include/mbedtls/rsa.h

```diff
@@ -718,7 +670,7 @@ int mbedtls_rsa_rsaes_oaep_encrypt( mbedtls_rsa_context *ctx,
  * \param f_rng    The RNG function. This is used for blinding and should
```

Comment:
There are still several functions where `f_rng` is documented as “should be provided” rather than mandatory.